ironfish/src/utils/bench.ts

```diff
@@ -55,13 +55,13 @@ function getSegment(): Segment {
     for (let i = 0; i < 5; ++i) {
       global.gc()
     }
-
-    const startMem = process.memoryUsage()
-    heap = startMem.heapUsed
-    rss = startMem.rss
-    mem = heap + rss
   }
 
+  const startMem = process.memoryUsage()
```

Comment:
Yeah but you can't measure memory if there's no GC because you can't collect to know how much was actually used. I think if there is no GC, memory measurements are not useful.